packages/i18n/src/index.js

```diff
@@ -1,63 +1,51 @@
 /**
  * External dependencies
  */
-import Jed from 'jed';
-import memoize from 'memize';
+import Tannin from 'tannin';
 
-let i18n;
+export { default as sprintf } from '@tannin/sprintf';
```

Comment:
I'm wondering if it wouldn't be preferable to simply use something like https://github.com/alexei/sprintf.js instead of introducing our own new library for this. `sprintf` seems like something that would already have a well-test well-maintained library.

Reply:
Agreed. Jed uses that library as well.

Reply:
It's an easy swap. I'd assumed `sprintf-js` to be quite bloated in its full-featuredness, though in truth it's fairly reasonable and efficient. It's still 6x the size of `@tannin/sprintf` (3x gzipped), though also includes functionality not supported by sprintf -- albeit the lesser-used functionality of printf patterns.

As far as implementation and performance, it [holds an internal cache](https://github.com/alexei/sprintf.js/blob/36b04b33975e597b03737c3ed0a2a49ec1b2e8c4/src/sprintf.js#L145-L147) for format strings, which could be argued as a minor memory leak. Without caching, they [perform similarly](https://gist.github.com/aduth/59b0e1b793bebb779883b4f754a1d6d3).

Maturity is a fair point to argue. In all, it's a fair swap I think.

Reply:
Swapped for `sprintf-js` in 9e07b809b905096f3de2a12008deaef8c696d20a